tmt/schemas/provision/virtual.yaml

```diff
@@ -0,0 +1,35 @@
+---
+
+#
+# JSON Schema definition for `virtual` provision plugin
+#
+# https://tmt.readthedocs.io/en/stable/spec/plans.html#virtual
+#
+
+$id: /schemas/provision/virtual
+$schema: https://json-schema.org/draft-07/schema
+
+type: object
+additionalProperties: false
+
+properties:
+
+  how:
+    type: string
+    enum:
+      - virtual
+
+  name:
+    type: string
+
+  image:
+    type: string
+
+  memory:
+    type: integer
+
+  user:
+    type: string
+
```

Comment:
Missing keys `disk`, `connection` and `arch`.

Reply:
@psss added, please check